pages/linux/factorio.md

```diff
@@ -0,0 +1,12 @@
+# factorio
+
+> Create and start a headless factorio server.
+> More information: <https://wiki.factorio.com/Multiplayer>.
+
+- Create a new save file:
+
+`{{path/to/}}factorio --create {{path/to/save_file.zip}}`
+
+- Start a factorio server:
```

Comment:
```suggestion
- Start a new Factorio server:
```

Reply:
I disagree with the word "new" as you can restart the server you closed yesterday.